xhr/overridemimetype-blob.html

```diff
@@ -6,60 +6,67 @@
 <div id="log"></div>
 <script>
 async_test(t => {
-  const client = new XMLHttpRequest()
+  const client = new XMLHttpRequest();
```

Comment:
This is extraordinarily hard to review because almost every line has changed, but I guess only because of semicolons? Please split up the commits at least next time.

Reply:
Sorry, I figured it wouldn't be much of an issue given how GitHub highlights stuff.